src/aposteriori/data_prep/create_frame_data_set.py

```diff
@@ -1165,11 +1174,54 @@ def _fetch_pdb(
     if len(pdb_code) == 5:
         # Extract chain from string:
         chain = pdb_code[-1]
-        _ = _select_pdb_chain(output_path, chain)
+        _, output_path = _select_pdb_chain(output_path, chain)
```

Comment:
I'm not keen on variable shadowing link this, it can lead to some annoying bugs, you might want to change the name to be more explicit, maybe `download_path`?

Reply:
It's meant to replace the path. This is bug what bug #38 was about. 

essentially if you want 1qysA, the code will download 1qys.pdb, select the chain and output it to 1qysA.pdb and then remove the original  1qys.pdb. If output_path isn't replaced by the new output_path with the chain it leads to an error. 

I could do if you prefer:

```
_, output_path_with_chain = _select_pdb_chain(output_path, chain)
output_path = output_path_with_chain
```

Reply:
How about having an else clause and 2 return statements with either `return base_output_path` and `return chain_output_path`? This is partly a stylistic thing, but I'm not keen on mutation like this, there's a good reason that lots of modern programming languages don't allow it: it's a common source of bugs.

Reply:
Yeah that could work. I'll fix it now